Templates/Specimens_Metadata_2021Q1_template.tsv

```diff
@@ -0,0 +1,18 @@
+Column Name	Description	Ontology identifier	Ontology term name	Ontology Description
+Specimen ID	Unique identifier for this specimen	OBI:0100051	specimen	A material entity that has the specimen role.
+Parent specimen ID	Identifier for this specimen’s parent specimen (optional)	OBI:0100051	specimen	A material entity that has the specimen role.
+Number of subspecimens	A count of specimens derived from this specimen (cumulative)	APOLLO_SV:00000032	count	A measurement datum that is the output of counting.
+Specimen type	Type of specimen (excised tissue block, slice, cell, etc.)	AIO:specimen type name	specimen type name	A textual entity that denotes the type of a specimen.
+Hemisphere	Hemisphere that this specimen came from (left, right, both, unknown)	PATO:0000140	position	A spatial quality inhering in a bearer by virtue of the bearer's spatial location relative to other objects in the vicinity.
```

Comment:
I might call this `Laterality` so that it is generalized outside the brain.

Reply:
Also (this may be quibbling, but), 'unknown' is a term that refers to epistemic status, not cerebral hemispheres.

Reply:
We see investigators use `unknown` when filling in sheets like this. Is there a better term? Some options that comes to mind would be `not reported` or `not measured` or `not recorded`. What is missing here is also `single hemisphere unspecified` as opposed to `might be left, right, or both`.

Reply:
My worry is that `hemisphere` is supposed to be take as a value a structure (or position? I'm not sure about `position` now that I'm thinking about it), not someone's knowledge/belief state. Even `not reported`, `not measured`, and `not recorded` are terms that refer to the (absence) of a process, not a structure. I think it could be a problem to allow these terms to mix here. If this is only supposed to be cerebral hemisphere, then maybe `left`, `right`, `both`, `neither`? I think that exhausts the possibilities.  <br/><br/> I will say, though, that `left` and `right` are directional qualities, while `both` and `neither` are not directional qualities, so this also wouldn't be technically correct. If the reporter doesn't know or cannot (with certainty) assert which hemisphere it came from, then it seems it should be left blank or `here's where the magic solution goes`. I'm happy to sign-off on whatever works for now, however.

Reply:
`neither` doesn't work though, because it was one or the other, it is just not know which one it was. `neither` would imply that it was some other anatomical part. Left blank doesn't work very well in these cases because there is no way to distinguish between the case where a user forgot to fill in the field, which results in an inability to distinguish between cases where it was know but not reported, and cases where it was not known at all. A similar issue may come up for the phenotypic sex field as well, though at least for BRAIN that is less likely to be the case due to the NIH focus on sex as a biological variable, there the issue is perhaps clearer because `(one-of '(male female))` -> `unknown` is clearly distinct from `(none-of '(male female))` -> `neither`.

Reply:
Right, I think we are in agreement that `neither` does not work. I don't think that `both` works either because `both` is not a hemisphere (in the same way that `neither` is not a hemisphere). Maybe `both` needs a superclass option -- `brain` or some such. Although, that would not quite convey the same information as `both`, I admit. 

> Left blank doesn't work very well in these cases because there is no way to distinguish between the case where a user forgot to fill in the field, which results in an inability to distinguish between cases where it was know but not reported, and cases where it was not known at all.

Exactly right, and it seems then that there is a temptation for this field to do two things: 1. specify a hemisphere where the specimen originates (when the user supplies that information), and 2. convey information about an epistemic state (in those other cases). If we provide the option of `unknown`, then the field becomes (rather explicitly) about the user's mental state, not brain hemispheres or anatomical structures. I don't think we are trying to track information about the user's epistemic state, so I want to try to eliminate the pull of (2) as much as we can. 

If the field is left blank, all we should infer is that the user left it blank (I think you agree). We should not infer anything about the epistemic state of the user from a blank field (for the reasons you give). I think this is consistent with the open-world assumption in OWL; although whether we want to make that assumption in this template is another question. 

Is there a third option for the field beside `brain` or would that work?

Reply:
Going up a level to `brain` works in some cases, but often it doesn't help you, e.g. if someone collects `thalamus` then we are back to where we started, was it the left thalams? right thalamus? a dissection that includes both? the grad student forgot to record which one it was? the laterality is blinded in the study? If they just report `thalamus` it is ambiguous and we know that we can't trust that a report of a lateralized structure means that both were collected (practically Uberon has usually avoided lateralizing structures).

There is yet another point here, which is that some structures aren't lateralized and would need something like `not relevant` or even the dreaded `not applicable`, the issue is that the collection form as implemented can't add or remove the `laterality` field depending on the anatomical structure so we wind up needing a place holder that users can fill in for case like that.I agree with points 1 and 2, but while we don't necessarily want to track (2) I do think users often want to convey that information to us, and scientifically it is important that we capture it, because if someone is studying e.g. gene expression that is lateralized then knowing whether a sample was collected from only one hemisphere as opposed to a sample that was pooled from both can be useful for segmenting data. So now I am up to conflating cardinality in addition to other things.

Ontologically bad, but practically I actually think it is ok, because all of the actual values that we would ever ask a user to deposit cleanly separate into disjoint types, which means that we can safely use a single cell in a spreadsheet to capture all of these different things. This means that transformation would have to be done on those types so that we know that there isn't a third `unknown` hemisphere, but because we can infer the implied type from the exact token that we are provided with, I don't think it is an issue.

Reply:
> Going up a level to brain works in some cases, but often it doesn't help you, e.g. if someone collects thalamus then we are back to where we started, was it the left thalams? right thalamus? a dissection that includes both? the grad student forgot to record which one it was? the laterality is blinded in the study? If they just report thalamus it is ambiguous and we know that we can't trust that a report of a lateralized structure means that both were collected (practically Uberon has usually avoided lateralizing structures)

Yes -- I agree that all of these are problems. I think part of this is due to ambiguity, part of this is due to a mismatch between supplied options and what actually occurred. I realize that `brain` is not a long-term solution.

> There is yet another point here, which is that some structures aren't lateralized and would need something like not relevant or even the dreaded not applicable, the issue is that the collection form as implemented can't add or remove the laterality field depending on the anatomical structure so we wind up needing a place holder that users can fill in for case like that.

Good point. I suppose the issue is what the placeholder will be in those cases. I suggest we choose something anatomical for consistency (or if it is `laterality`, then maybe `bilateral` or whatever makes sense). 

> Ontologically bad, but practically I actually think it is ok

Exactly. I'm fine signing-off on this as @kasbaker has it for now. I just wanted to point out that our ontology will not have this modeled like this and it's not precisely correct (I admit that it was a quibbling point). However, if it works for the submitters and data folks, that's fine.

Reply:
So I think we are really talking about `position` (`PATO:0000140`) relative to the midline. Position has subclasses like `right side of` and `left side of`. I can't find an ontology entry for something like `midline though`.

Reply:
http://purl.obolibrary.org/obo/BSPO_0000009 `midsagittal plane` seems like what you are looking for, that is part of an operational definition for left and right though, which may not always be available, there might be left hand or right hand which are determined by the position of the thub independent of any reference to the midline.

Reply:
@tgbugs I think that's correct. Could use http://purl.obolibrary.org/obo/BSPO_0000007 `right side` and http://purl.obolibrary.org/obo/BSPO_0000000 `left side` in this case (they are defined via `sagittal plane` in BSPO). 

The hand case is a special case that I don't want to touch (pun intended).

Reply:
I changed hemisphere to two entries `Has origin left side` and `Has origin right side` which will take true/false values.